rasa/cli/test.py

```diff
@@ -83,20 +83,43 @@ def test_core(args: argparse.Namespace) -> None:
         )
 
     else:
-        test_compare(args.model, stories, output)
+        test_compare_core(args.model, stories, output)
 
 
 def test_nlu(args: argparse.Namespace) -> None:
     from rasa.test import test_nlu, test_nlu_with_cross_validation
+    import os
 
     nlu_data = get_validated_path(args.nlu, "nlu", DEFAULT_DATA_PATH)
     nlu_data = data.get_nlu_directory(nlu_data)
 
-    if not args.cross_validation:
+    if isinstance(args.config, list):
+        if len(args.config) == 1:
+            args.config = args.config[0]
+            config_path = os.path.abspath(args.config)
+
+            if os.path.isdir(config_path):
+                configs = os.listdir(args.config)
+                args.config = [os.path.join(config_path, config) for config in configs]
+
+    if isinstance(args.config, list):
+        logger.info("Multiple configs specified, running nlu comparison mode.")
+
+        output = args.report or "nlu_comparison_results"
+        test_compare_nlu(
+            configs=args.config,
+            nlu=nlu_data,
+            output=output,
+            runs=args.runs,
+            exclusion_percentages=args.percentages,
+        )
+
+    elif not args.cross_validation:
```

Comment:
Could you change the order so that it is more readable?
```
    elif args.cross_validation:
        logger.info("Test model using cross validation.")
        config = get_validated_path(args.config, "config", DEFAULT_CONFIG_PATH)
    else:
        model_path = get_validated_path(args.model, "model", DEFAULT_MODELS_PATH)
        test_nlu(model_path, nlu_data, vars(args))
```